src/server/controllers/s3.js

```diff
@@ -4,6 +4,7 @@ const S3 = require('aws-sdk/clients/s3')
 
 const defaultConfig = {
   acl: 'public-read',
+  endpoint: 'https://{service}.{region}.amazonaws.com',
```

Comment:
should this be a template string?

Reply:
No, the AWS SDK fills in the `{service}` and `{region}` things internally